driver-core/src/main/java/com/datastax/driver/core/querybuilder/QueryBuilder.java

```diff
@@ -228,7 +229,7 @@ public static Truncate truncate(TableMetadata table) {
      * @return the quoted column name.
      */
     public static String quote(String columnName) {
```

Comment:
@al3xandru drew my attention to this method recently; I suggest that we delegate to `Metadata.quote(String)` to avoid code duplication/entropy.